<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7ARKDEF" failures="2" tests="9" name="RetryAnalyzerTestCases" time="0.186" errors="0" timestamp="2020-12-27T13:25:48 EST">
  <testcase classname="com.test.MyTest" name="test3" time="0.017">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="com.test.MyTest" name="test3" time="0.002">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="com.test.MyTest" name="test3" time="0.005">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="com.test.MyTest" name="test3" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.MyTest.test3(MyTest.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="com.test.MyTest" name="test4" time="0.009">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase classname="com.test.MyTest" name="test4" time="0.003">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase classname="com.test.MyTest" name="test4" time="0.008">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase classname="com.test.MyTest" name="test4" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.MyTest.test4(MyTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase classname="com.test.MyTest" name="test5" time="0.012"/>
</testsuite> <!-- RetryAnalyzerTestCases -->
